### PR TITLE
Deprecated videoanalyzer and retract v0.4.1

### PR DESCRIPTION
**Pull request by Alancere**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by tadelesh** [comment]
Wait for final deprecation message.

File: sdk/resourcemanager/videoanalyzer/armvideoanalyzer/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.4.2 (2022-05-16)
+### Deprecated
+-
+
 ## 0.4.1 (2022-05-07)
 ### Deprecated
 - We’re retiring the Azure Video Analyzer preview service, you're advised to transition your applications off of Video Analyzer by 01 December 2022. This SDK is no longer maintained and won’t work after the service is retired.
```
Comment (tadelesh) [inline sdk/resourcemanager/videoanalyzer/armvideoanalyzer/CHANGELOG.md:9]: Remove this part as the version has retracted.

Comment (Alancere) [reply sdk/resourcemanager/videoanalyzer/armvideoanalyzer/CHANGELOG.md]: removed

**Comment by tadelesh** [comment]
please also deprecate packages under profiles

**Review by tadelesh — APPROVED** [review]

**Pull request merged**